packages/sdk/src/errors.ts

```diff
@@ -0,0 +1,50 @@
+export class NeonError extends Error {
+
+  constructor(message: string) {
+    super(message);
+
+    if (Error.captureStackTrace) {
```

Comment:
You don't need to do this. Stack traces are automatically added when the error is created